adobe-photoshop/adobe-photoshop-quiz-fr.md

```diff
@@ -551,3 +551,17 @@
 #### Q73. Pour des bords incurvés plus lisses sur une sélection, quelle sélection devriez-vous choisir ?
 
 \-\[x] anticrénelage
+
+#### Q74. Which tool is *not* found in the **Select and Mask** workspace ?
+
+- \[ ] Magic Wand
+- \[x] Refine Edge
+- \[ ] Quick Select
+- \[ ] Lasso
+
+#### Q75. Which sets of tool has a content-aware mode ?
+
+- \[ ] Perspective Corp
+- \[ ] Marquee
+- \[ ] Lasso
+- \[x] Patch
```

Comment:
@gitsdeepak thanks a lot for your PR, but could you please move those question under different file where all question has English version, right now you are in Italy version.
Thanks a lot in advance! if you want close this PR and create new one.

Reply:
Sure